lib/model/query/forms.js

```diff
@@ -258,6 +262,13 @@ with redacted_audits as (
     left outer join form_defs on coalesce(forms."currentDefId", forms."draftDefId") = form_defs.id
     where actees.id = forms."acteeId"
     and ${_trashedFilter(force, id, projectId)}
+  ), update_entities as (
```

Comment:
if we change the migration above to "on delete set null", then wont this just happen automatically?